pkg/component/plutono/dashboards/shoot/owners/kubernetes-pods-dashboard.json

```diff
@@ -828,6 +828,6 @@
   },
   "timezone": "utc",
   "title": "Kubernetes Pods",
-  "uid": "kube-pods",
+  "uid": "shoot-kube-pods",
```

Comment:
Regarding the failing unit tests: you need to execute `pkg/component/plutono/plutono_suite_test.go` and "extract" the new generated hash for the configmap name e.g. by debugging the failing test. If the dashboards are changed, the configmap suffix in the unit tests needs to be adjusted, like here:

https://github.com/gardener/gardener/pull/8390

Reply:
Just to note the result of a call with @Sallyan:

> To see the new hashes I added:
>```
>configmaps := &corev1.ConfigMapList{}
>c.List(ctx, configmaps)
>```
> and
>```
>(*configmaps).Items[0].ObjectMeta.Name
>```
>to my watches.

which I found a more convenient way to access the hashes. The change in `pkg/component/plutono/plutono_suite_test.go` therefore needs to be as follows:

```
plutono-dashboards-garden-fbef06ba -> plutono-dashboards-garden-40b40715
plutono-dashboards-4f58da97 -> plutono-dashboards-6938ad4b
plutono-dashboards-e60246de -> plutono-dashboards-72c159d8
plutono-dashboards-bc6281a9 -> plutono-dashboards-d72a9abb
```

And for the record, I find this test problematic. I understand it doesn't specifically aim to test the names, but it's both cumbersome and going against the purpose of a test to have to copy values from the debugger in order to add them to the code.